src/elastio-snap.c

```diff
@@ -111,6 +111,12 @@ struct path {
 #define elastio_snap_get_mnt(f) (f)->f_path.mnt
 #endif
 
+#ifdef HAVE_BDEVNAME
+#define elastio_snap_bdevname(dev, buf) bdevname(dev, buf)
+#else
+#define elastio_snap_bdevname(dev, buf) snprintf(buf, sizeof(buf), "%pg", dev);
+#endif
```

Comment:
maybe just 
```suggestion
#ifndef HAVE_BDEVNAME
#define bdevname(dev, buf) snprintf(buf, sizeof(buf), "%pg", dev);
#endif
```
?

Reply:
I don't think so. I wish the person to understand this is not a default `bdevname`. This should be explicitly mentioned.

Reply:
ok